VulnerableApplication/Global.asax.cs

```diff
@@ -17,5 +17,7 @@ protected void Application_Start()
             RouteConfig.RegisterRoutes(RouteTable.Routes);
             BundleConfig.RegisterBundles(BundleTable.Bundles);
         }
+        
+        public const string Password = "abc";
```

Comment:
**CODE_SMELL**
> Add or update the header of this file.
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWure2BHojvkCqRLlWU7&pullRequest=16&resolved=false)

Reply:
/falsepositive

Reply:
The issue marked as falsepositive. For more [detail](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWure2BHojvkCqRLlWU7&pullRequest=16).